source/server/config/http/ext_authz.cc

```diff
@@ -0,0 +1,66 @@
+#include "server/config/http/ext_authz.h"
+
+#include <chrono>
+#include <string>
+
+#include "envoy/config/filter/http/ext_authz/v2/ext_authz.pb.validate.h"
+#include "envoy/registry/registry.h"
+
+#include "common/ext_authz/ext_authz_impl.h"
+#include "common/http/filter/ext_authz.h"
+#include "common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+HttpFilterFactoryCb ExtAuthzFilterConfig::createFilter(
+    const envoy::config::filter::http::ext_authz::v2::ExtAuthz& proto_config, const std::string&,
+    FactoryContext& context) {
+
+  ASSERT((proto_config.grpc_service().has_envoy_grpc() &&
```

Comment:
Does this really help? Ideally this and https://github.com/envoyproxy/envoy/blob/master/source/server/config/network/ext_authz.cc are as similar as possible.